[HANDLOWY]supplier form

JustynaKras opened the pull request.

JustynaKras committed:
Merge branch 'feature/HANDLOWY/OutsourcingForm' of https://github.com/rudnickirobert/KWZP2019

JustynaKras committed:
[HANDLOWY] add new supplier

JustynaKras committed:
[HANDLOWY]Changes in SupplieForm

rudnickirobert commented on KWZP2019/KWZP2019/AddNewOutsourcingCompanyForm.cs:
```diff
@@ -12,9 +12,106 @@ namespace KWZP2019
+            {
+                MessageBox.Show("Nazwa jest wymagana");
+            }
+
```
po co pusta linia?

rudnickirobert replied:
zamiast else można dać `return` po pokazaniu MessageBoxa

rudnickirobert commented on KWZP2019/KWZP2019/AddNewOutsourcingCompanyForm.cs:
```diff
@@ -12,9 +12,106 @@ namespace KWZP2019
+                }
+                else
+                {
+                    MessageBox.Show("Niewłaściwy format numeru domu");
```
pokażemy komunikat ale kod dalej się wykona- chyba tak nie powinno być?

rudnickirobert commented on KWZP2019/KWZP2019/AddNewSupplierForm.cs:
```diff
@@ -12,9 +12,108 @@ namespace KWZP2019
+        private void addNewSupplierBtn_Click(object sender, EventArgs e)
+        {
+            //ŻĄDANIE WPROWADZENIA NAZWY FIRMY 
+            if (nameSupplierTb.Text.Trim() == "")
```
jw

Maciej44Kowalski committed:
Wprowadzone poprawki

rudnickirobert reviewed (changes requested):
konflikty

Maciej44Kowalski committed:
[HANDLOWY] Rozwiązanie kofliktów; dostosowanie do nowego GUI z brancha master; naprawa EF

Merge branch 'master' of https://github.com/rudnickirobert/KWZP2019 into feature/HANDLOWY/SupplierForm

rudnickirobert commented on KWZP2019/KWZP2019/AddNewCustomerForm.cs:
```diff
@@ -28,77 +28,77 @@ private void addNewCustomerBtn_Click(object sender, EventArgs e)
+            }
+            Customer newCustomer = new Customer();
+            newCustomer.CustomerName = nameTb.Text.Trim();
+            bool phoneTb2 = int.TryParse(phoneTb.Text, out int phoneTb21);
```
dlaczego `Tb2` i `Tb21`?

rudnickirobert commented on KWZP2019/KWZP2019/SupplierForm.cs:
```diff
@@ -12,15 +12,47 @@ namespace KWZP2019
+        private void Dostawcapolfabrykatu_rb_CheckedChanged(object sender, EventArgs e)
+        {
+            
+            if (Dostawcapolfabrykatu_rb.Checked)
```
nazwa checkboxa powinna byc camelCase

rudnickirobert reviewed (approved):
@JustynaKras wmergowane ale prosze poprawic uwagi (w nowym PR)
- https://github.com/rudnickirobert/KWZP2019/pull/89#discussion_r281071930
- https://github.com/rudnickirobert/KWZP2019/pull/89#discussion_r281072144
